### PR TITLE
winsparkle: update to 0.7.0

### PR DESCRIPTION
**Pull request by raedrizqie**

### CODE REVIEW

File: mingw-w64-winsparkle/001-compile-fixes.patch [hunk 1]
```diff
@@ -1,11 +1,114 @@
...
++	${CXX} -shared -o ${DLLNAME} ${OBJS} ${RESFILE} -Wl,--out-implib,$@ ${LDFLAGS} ${WX_LIBS} ${SSL_LIBS} ${LIBS}
+diff -urN winsparkle-0.7.0-orig/src/dll_api.cpp winsparkle-0.7.0/src/dll_api.cpp
+--- winsparkle-0.7.0-orig/src/dll_api.cpp	2020-06-07 16:50:21 +0800
++++ winsparkle-0.7.0/src/dll_api.cpp	2021-11-04 17:20:21 +0800
+@@ -273,7 +273,7 @@
+     return DEFAULT_CHECK_INTERVAL;
+ }
+ 
+-WIN_SPARKLE_API time_t __cdecl win_sparkle_get_last_check_time()
++WIN_SPARKLE_API size_t __cdecl win_sparkle_get_last_check_time()
```
Comment (lazka) [inline mingw-w64-winsparkle/001-compile-fixes.patch:88]: Why the time_t/size_t switch?

Comment (raedrizqie) [reply mingw-w64-winsparkle/001-compile-fixes.patch:88]: failed on clang.. conflicting types

Comment (lazka) [reply mingw-w64-winsparkle/001-compile-fixes.patch:88]: hm, but it's not used anywhere.

Comment (raedrizqie) [reply mingw-w64-winsparkle/001-compile-fixes.patch]: fixed now..

**Pull request closed without merging**